src/pages/verify/guides/telemetry.mdx

```diff
@@ -0,0 +1,46 @@
+---
+product: verify
+category: Guides & Tools
+sort: 10
+title: Telemetry & observability
+---
+
+Criipto offers telemetry and observability into authentication flows based on OpenTelemetry.
```

Comment:
Does it matter to our customers that it's based on OpenTelemetry? This makes it seem like they can ingest the data somehow, which AFAIK they can't.

Reply:
cc @kasperhj

Reply:
b9c70ba